js/index.js

```diff
@@ -1,6 +1,94 @@
-const startApp = () => {
-  const body = document.querySelector('body');
-  body.innerHTML = '<h2>JAVASCRIPT ENABLED</h2>';
+const form = document.querySelector('#form');
+const btn = document.querySelector('#add-btn');
+const libraryList = document.querySelector('#library-list');
+
+let myLibrary = [];
+
+const getID = () => {
+  const id = new Date()
+    .getTime()
+    .toString(36)
+    .concat(performance.now().toString(), Math.random().toString())
+    .replace(/\./g, '');
+  return id;
+};
+
+const clear = () => {
+  form.Author.value = '';
+  form.Title.value = '';
+};
+
+// saving to local
+const addBookToLocal = myLibrary => {
+  localStorage.setItem('myLibrary', JSON.stringify(myLibrary));
+};
+
+const createBook = () => {
+  const book = { id: '', title: '', author: '' };
+  book.id = getID();
+  book.title = form.Title.value;
+  book.author = form.Author.value;
+  myLibrary.push(book);
+  myLibrary.join(', ');
```

Comment:
Is there any reason why we need this join function here, shouldn't the `push` be enough?

Reply:
code adjusted accordingly